src/main/java/rx/Completable.java

```diff
@@ -1843,7 +1846,7 @@ public final Completable startWith(Completable other) {
      */
     public final Subscription subscribe() {
         final MultipleAssignmentSubscription mad = new MultipleAssignmentSubscription();
-        subscribe(new CompletableSubscriber() {
+        unsafeSubscribe(new CompletableSubscriber() {
```

Comment:
You're right. However, the other 2 `subscribe` methods that don't use `SafeCompletableSubscriber` should do it. To assert that the `Action` instances are called at most once, and not both, complying with the contract.